test/buildah-bud/buildah-tests.diff

```diff
@@ -108,19 +95,6 @@ index 166316d5..775d7c9b 100644
          # without "quotes", multiple lines are glommed together into one
          if [ -n "$output" ]; then
              echo "$output"
-@@ -477,3 +531,12 @@ function skip_if_no_docker() {
-     skip "this test needs actual docker, not podman-docker"
-   fi
- }
-+
-+####################
-+#  skip_if_remote  #  (only applicable for podman)
-+####################
-+function skip_if_remote() {
-+    if [[ -n "$REMOTE" ]]; then
-+        skip "${1:-test does not work with podman-remote}"
-+    fi
-+}
```

Comment:
I'm pretty sure that removing this is going to cause CI to fail. Do you remember why you removed this?